Tasks/XamarinTestCloud/task.json

```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 126,
+        "Minor": 127,
```

Comment:
We use a convention to set it to the current VSTS sprint which is 129. Would be easier to track down changes if you followed that.

Reply:
There will be a .loc json file generated when you build, please push that as well.